components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -78,22 +112,36 @@ export default class ChannelIntroMessage extends React.PureComponent<Props> {
             centeredIntro = 'channel-intro--centered';
         }
 
+        let classes: string;
+        const createClasses = (withSvg: boolean) => {
+            if (this.props.theme?.type === 'Onyx' && !withSvg) {
+                classes = 'channelIntroButton';
+                return classes;
+            } else if (this.props.theme?.type === 'Onyx' && withSvg) {
+                classes = 'channelIntroButton withSvg';
+                return classes;
+            } else if (!withSvg) {
+                classes = 'channelIntroButton';
+                return classes;
+            }
+            classes = 'channelIntroButton withSvg';
+            return classes;
+        };
```

Comment:
instead of a clunky `createClass` function please use the `classname` package

in a `const`
```
const classes = classNames('channelIntroButton', {withSvg});
```

or directly in the props of a componet
```
<Component className={classNames('channelIntroButton', {withSvg}}/>
```